google/resource_compute_shared_vpc_test.go

```diff
@@ -0,0 +1,129 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"os"
+)
+
+func TestAccComputeSharedVpc_basic(t *testing.T) {
```

Comment:
Can you also check that delete works? https://github.com/terraform-providers/terraform-provider-google/pull/396/commits/5f9ef48338d46fa57680c1e84af19c2e9eadfe08 may be of use here

Reply:
Done